lib/executable-config/lookup/src-ghcjs/Obelisk/ExecutableConfig/Lookup.hs

```diff
@@ -35,7 +35,7 @@ getConfigs = do
   where
     collToList es = do
       len <- getLength es
-      list <- traverse (item es) [0..len-1]
+      list <- traverse (item es) $ if len == 0 then [] else [0..len-1]
```

Comment:
And this, folks, is why subtraction is not a total function.

Reply:
Super nitpick idea: `[0..max 0 (len-1)]`

Reply:
That actually behaves a little differently when `len = 0`

Reply:
Aha right I forgot it was inclusive.

Reply:
Haha.... wow. It's hard to deal with under/over flow! If subtraction were `Word -> Word -> Maybe Word` then `max 0 (len-1)` wouldn't type check either. Super tricky.

Reply:
I usually just avoid unsigned types like the plague for this reason. I'm just curious why we aren't.